diff -ruN apache_airflow_providers_segment-3.8.0/pyproject.toml apache_airflow_providers_segment-3.8.0-patch/pyproject.toml
--- apache_airflow_providers_segment-3.8.0/pyproject.toml	2025-05-13 12:54:24.000000000 +0200
+++ apache_airflow_providers_segment-3.8.0-patch/pyproject.toml	2025-05-19 15:40:41.434854503 +0200
@@ -58,7 +58,7 @@
 # After you modify the dependencies, and rebuild your Breeze CI image with ``breeze ci-image build``
 dependencies = [
     "apache-airflow>=2.10.0",
-    "analytics-python>=1.2.9",
+    "segment-analytics-python>=1.2.9",
 ]
 
 [dependency-groups]
